runtime.md

```diff
@@ -15,9 +15,9 @@ This MUST be unique across all containers on this host.
 There is no requirement that it be unique across hosts.
 * **`status`**: (string) is the runtime state of the container.
 The value MAY be one of:
-    * `created`: the container has been created but the user-specified code has not yet been executed
-    * `running`: the container has been created and the user-specified code is running
-    * `stopped`: the container has been created and the user-specified code has been executed but is no longer running
+    * `created`: the container process has neither exited nor executed the user-specified code
+    * `running`: the container process has executed the user-specified code but has not exited
+    * `stopped`: the container process has exited
```

Comment:
Maybe the container process crashed during creation, in which case we're `stopped` before the container has been fully created.  And see #507, which builds on this PR and adds `creating`.  In the absence of a `creating` status (this PR before #507), I think we're `created` even before creation has finished.